src/components/uiOptions/js/UIOptions.js

```diff
@@ -26,4 +26,38 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         ]
     });
 
+    /** A configuration of UIOptions which is suitable for statically localised contexts. It accepts two top-level
+     * options,
+     * {String} locale - The initial locale in which UIOptions should render
+     * {String} direction - A suitable value for the `dir` attribute of the UIOptions container - this may take
+     * values `ltr`, `rtl` or `auto`
+     */
+    fluid.defaults("fluid.uiOptions.multilingual", {
+        gradeNames: ["fluid.uiOptions"],
+        distributeOptions: {
+            source: "{that}.options.locale",
+            target: "{that messageLoader}.options.resourceOptions.locale"
+        },
+        listeners: {
+            "onPrefsEditorReady.addLanguageAttributesToBody": {
+                func: "fluid.uiOptions.multilingual.addLanguageAttributesToBody",
+                args: ["{that}.prefsEditorLoader.prefsEditor.container", "{that}.options.locale", "{that}.options.direction"],
```

Comment:
This can be done declaratively using `this` as the container is a jQuery element already and the `attr` method can take an object of attributes to apply. See: https://github.com/fluid-project/trivet/pull/148#discussion_r652754843